app/conf/train-with-me/interactionManager.ini

```diff
@@ -13,3 +13,5 @@ nrep-perform              16
 virtual-mode              false
 engage-with-hand          false
 wait-for-imitation        true
+use-robot-template        false
```

Comment:
I'd like to see the robot template mode as the default 😉 
Also, in other conf files, this flag is already true.

Reply:
I put the flag as false only in the `train-with-me` conf file, because we used the pre-recorded templates during the experiments. Do you think the flag should be true also in the `train-with-me` conf file?

Reply:
Ah! Got it!
Anyway, it might be worth switching to the robot's template also in `train-with-me` mode.
What do you thing?

Reply:
I think that this should certainly be the default mode, so I'm okay to switch to this in the `train-with-me` mode, also in view of future experiments.